### PR TITLE
Design qa

### PR DESCRIPTION
**Pull request by itsjoekent**
### What does this PR do?
1. Fixes the reportback item going extra wide if the figure body is one continuous word (eg: a youtube link)

screen shot 2017-09-27 at 10 19 05 am

2. Fixes the byline so it displays inline with the share

screen shot 2017-09-27 at 10 19 24 am

3. Fixes the Share hover effect

### CODE REVIEW

**Comment by itsjoekent** [comment]
oh shiz wait, wrong word break haha

**Comment by itsjoekent** [comment]
better 
screen shot 2017-09-27 at 10 24 54 am

File: resources/assets/components/Share/share.scss [hunk 1]
```diff
@@ -23,6 +23,7 @@
     &:hover {
       color: $med-gray;
       cursor: pointer;
+      background: none;
```
Comment (weerd) [inline resources/assets/components/Share/share.scss]: why was this removed? just curious

Comment (itsjoekent) [reply resources/assets/components/Share/share.scss]: hover effect without it

screen shot 2017-09-27 at 10 29 49 am

speaking of which, i should remove it for the other hover states as well
